### ISSUE TITLE
Disable es-x-no-symbol-prototype-description

### ISSUE THREAD

**Issue by edg2s** [wikimedia/jscs-preset-wikimedia#477]
Per https://ota-meshi.github.io/eslint-plugin-es-x/rules/no-symbol-prototype-description.html#es-x-no-symbol-prototype-description:
> This rule is silent by default because it's hard to know types. You need to configure [the aggressive mode](https://ota-meshi.github.io/eslint-plugin-es-x/#the-aggressive-mode) or TypeScript in order to enable this rule.

Which means this triggers many false positives on any object property called `description`.

Reported offline by @matmarex

**Comment by edg2s** [comment]
We may want to audit the other prototype rules. I think the only one we disable at the moment is `find` due to conflict with `jQuery.find`, but there may be others with names that are generic enough to trigger too many false positives.

**Comment by MatmaRex** [comment]
I was also negatively surprised by `es-x/no-array-prototype-keys` and `es-x/no-array-prototype-values`. I learned about the changes when I noticed noticed all three being disabled in <https://gerrit.wikimedia.org/r/c/mediawiki/extensions/UploadWizard/+/838537>.

In particular, `es-x/no-array-prototype-keys` is triggered by `Object.keys( foo )`, which is obviously not calling the method being disallowed.

Back in the day, when we first disallowed some methods in 57b2fdc5ae47af43f4cebc36fb00ea7a9b1cf815, we were actually smart enough to disallow `foo.keys()` without disallowing `Object.keys( foo )`: https://github.com/wikimedia/eslint-config-wikimedia/commit/57b2fdc5ae47af43f4cebc36fb00ea7a9b1cf815#diff-994979481b5e2ff41767c0b7f3485267ce7402a2c0f9a66945ff2785959927a0R88-R91 so I'm hopeful that this kind of solution can be restored.
